frontend/i18n/pl/home.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  mainTitle: 'Adnotacja tekstowa dla ludzi',
+  getStarted: 'Rozpocznij',
```

Comment:
Obviously I am not fluent in Polish either, but I am pretty sure that instead of `adnotacj_` (adnotacja, but the ending can be conjugated so--), the word `anotacj_` is more precise for this. I searched in the discord server and couldn't find "adnotacji", but I could find "anotacji" several times. Also: https://dobryslownik.pl/slowo/anotacja/224215/

So imo we should replace `adnotacj_` with `anotacj_` 😅